tests/providers/google/suite/operators/test_sheets_system.py

```diff
@@ -17,8 +17,8 @@
 
 import pytest
```

Comment:
In the new design this file is not required. Please ensure that the setup/teardown fixtures are migrated to the DAG itself.